### PR TITLE
media server docker container

### PR DESCRIPTION
**Pull request by hamsterksu**
@deruelle the first version of container

### CODE REVIEW

**Comment by deruelle** [comment]
Thanks @hamsterksu ! 
@hrosa @leftyb can you please review ?

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+FROM java:8-jdk
```
Comment (hrosa) [inline Dockerfile]: Official supported version is 7

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+ENV work_dir /opt/restcomm-media-server
+
+RUN unzip /tmp/restcomm-media-server.zip -d /opt && \
+    chmod +x $work_dir/bin/run.sh && \
+    rm /tmp/restcomm-media-server.zip && \
+    mv /tmp/media-version.txt $work_dir
+
+ADD scripts/configure.sh $work_dir
+ADD scripts/run.sh $work_dir
```
Comment (hrosa) [inline Dockerfile:20]: Are these auto-configures scripts that you created? Similar to the ones we have for RestComm?
If so, they should be included with the binary instead.

Comment (hamsterksu) [inline Dockerfile:20]: i have created it. names are similar.

**Comment by hrosa** [comment]
@hamsterksu @deruelle commented inline in the commit

**Comment by hamsterksu** [comment]
@hrosa i have updated java version.

**Pull request merged**